GDJS/Runtime/howler-sound-manager/howler-sound-manager.js

```diff
@@ -240,20 +303,79 @@ gdjs.HowlerSoundManager.prototype._storeSoundInArray = function (arr, sound) {
   return sound;
 };
 
+/**
+ * Creates a new gdjs.HowlerSound using preloaded/cached Howl instances.
+ * @param {string} soundName The name of the file or resource to play.
+ * @param {boolean} isMusic True if a music, false if a sound.
+ * @returns {gdjs.HowlerSound}
+ */
+gdjs.HowlerSoundManager.prototype.createHowlerSound = function (soundName, isMusic) {
+  var soundFile = this._getFileFromSoundName(soundName);
+  var container = this._loaded[isMusic ? "music" : "sound"];
+
+  if (
+    container.hasOwnProperty(soundFile)
+  ) {
+    return new gdjs.HowlerSound(container[soundFile]);
```

Comment:
Does this work if you run multiple time the sound? Like an explosion effect, which can be run multiple times at the same time.